src/actors/ball.ts

```diff
@@ -0,0 +1,61 @@
+import * as ex from 'excalibur';
+import {GameEvent} from "excalibur";
```

Comment:
Used single quotes for string above, then double quotes here. Single quotes here I'd say